deb/openmediavault/srv/salt/omv/deploy/apt/default.sls

```diff
@@ -20,6 +20,7 @@
 {% set config = salt['omv_conf.get']('conf.system.apt.distribution') %}
 {% set use_kernel_backports = salt['pillar.get']('default:OMV_APT_USE_KERNEL_BACKPORTS', True) -%}
 {% set proxy_config = salt['omv_conf.get']('conf.system.network.proxy') %}
+{% set rpi = salt['pillar.get']('default:OMV_IS_RPI', False) %}
```

Comment:
Could we name this variable like this? Then it is equal to `OMV_APT_USE_KERNEL_BACKPORTS`.
```suggestion
{% set use_os_security = salt['pillar.get']('default:OMV_APT_USE_OS_SECURITY', False) %}
```